.github/copilot-instructions.md

```
@@ -0,0 +1,295 @@
+| HTTP | axios |
+| 스타일 | **CSS Modules (SCSS)** + `clsx` |
+| 테스트 | Vitest + React Testing Library |
+| 패키지 매니저 | **pnpm** |
+
+Tailwind 를 쓰지 않습니다. styled-components 도 쓰지 않습니다.
+
+---
+
+## 디렉터리 구조
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)